### DISCUSSION TITLE
type of search [voting]

### DISCUSSION THREAD

**Discussion by amd64fox** [SpotX-Official/SpotX#46] [Polls]
**New** | **Old**
:----: | :----:
![Снимок экрана 2022-05-23 130811](https://user-images.githubusercontent.com/62529699/169799460-edfb5f00-65c3-4c3d-8810-fd47abec1ede.jpg)  | ![Снимок экрана 2022-05-23 131155](https://user-images.githubusercontent.com/62529699/169799510-1a95d220-f416-40cd-af52-bf47a60e0566.jpg)

**Comment by JasonShearer** [comment]
If possible, maybe an option for both?

[Edit: To clarify, I mean an option to select between either. Not a combination of the two.]

**Reply by amd64fox** [reply]
It will be too cumbersome if you add an optional selection for such small functions during installation.

You can always adjust this moment for yourself manually, [here is a brief instruction](https://github.com/amd64fox/SpotX/discussions/47).
